### ISSUE TITLE
OCR integration with camera

### ISSUE THREAD

**Issue by tinkerofwales** [tinkerofwales/tuna_munchers#5]
Modify the camera app to interface and display the results of passing the image through the OCR api

**Comment by radjaffo** [comment]
OCR works on my windows desktop using Eclipse, running into issues with NDK and Android Studio.

**Comment by radjaffo** [comment]
So i got a basic test case up of the functional OCR early this morning, however I'm really not happy with the results... It is extremely hit or miss
I took some sample pictures using a textbook, a test, and my computer screen.
main problems encountered were.
If the light fades at all near the bottom of the picture it chooses what it wants to capture then fills in the rest with nonsense
If there is a bounding box it has problems, it seems to think the box is text and tries to translate it to no avail.
Math equations are all over the place, as well as graphs.
If the picture is crooked it throws all sorts of nonsense into it.
I think has something to do with picture resolution, light, and picture orientation. Orientation should be an easy fix, not sure on the other two...
Setting it up is a minor pain but I understand it very well at this point and can "easily" implement it into our exisiting app.

If you want to test it on your environments let me know and I'll give a quick explanation of how I got it up and running.
